configs/BARTpre.yaml

```
@@ -0,0 +1,29 @@
+train:
+  num_epochs: 5
+  accum_iter: 10
+  batch_size: 8
+  base_lr: 0.1
+  warmup: 4000
+
+model_name: BART
+dataset_name: 'roszcz/maestro-v1-sustain'
+target: mask-filling
+seed: 26
+
+overfit: False
+masking_probability: 0.3
+dataset:
+  sequence_duration: 2
+  sequence_step: 2
+
+  quantization:
+    start: 20
+    duration: 3
+    velocity: 3
+
+device: "cpu"
+
+log: False
```

```suggestion
log: True
```

I think we should log everything by default